<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="btn_a">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_w">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_y">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_u">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>U</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_t">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>T</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_s">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_r">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_h">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>150</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_o">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>O</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_n">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_m">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>M</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_l">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_p">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_k">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>K</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_v">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_j">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>150</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>J</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_i">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>I</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_q">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_x">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>250</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_g">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>G</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_f">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_e">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_d">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_c">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_b">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_z">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
font: 700 italic 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>571</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 50pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 127);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_h">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>              Choose the letters you want</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
